src/components/AuthWrapper.jsx

```diff
@@ -4,6 +4,7 @@ import LanguageDropdown from './LanguageDropdown'
 import eosIcon from '../assets/images/user-story-logo.svg'
 import SocialMediaLinks from '../components/SocialMediaLinks'
 import { EOS_COPYRIGHT } from 'eos-icons-react'
+import { Link } from '@reach/router'
```

Comment:
this issue seems to already have been addressed in https://github.com/EOS-uiux-Solutions/user-story/pull/162

Please remove these changes from this PR so that the other one can be merged